terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1201,6 +1201,13 @@ def qq(*words):
                 'iam_instance_profile': '${aws_iam_instance_profile.gitlab.name}',
                 'ami': ami_id[config.region],
                 'instance_type': 't3a.xlarge',
+                'metadata_options': {
+                    'http_tokens': 'required',
+                    # This value was determined by running a pipeline in GitLab
+                    # after updating this field with different values. Using 2
+                    # failed the pipeline, 3 succeeded and informed the selection.
+                    'http_put_response_hop_limit': 3
```

Comment:
I would rewrite this to be more terse and also mention how the pipeline failed (what error was observed?) when the value was reduced to 2.

```suggestion
                    # Empirically determined minimum value for a pipeline to succeed. Using a lower value resulted in ... (describe failure)
                    'http_put_response_hop_limit': 3
```

Reply:
Opted for a comment that Hannes suggested.